src/main/java/prefs/Preferences.java

```diff
@@ -15,23 +15,30 @@
  * It cannot be as we need to mock it.
  */
 public class Preferences { // NOPMD
+
     public static final String DIRECTORY = "settings";
 
     // Standard config filenames used for application and testing
-    public static final String GLOBAL_CONFIG_FILE = "global.json";
-    public static final String TEST_CONFIG_FILE = "test.json";
+    public static final String SESSION_CONFIG_FILENAME = "global.json";
+    public static final String TEST_SESSION_CONFIG_FILENAME = "test.json";
```

Comment:
Filename should be an instance level variables fed to the constructor. That allows using different files for testing and production.